src/wp-content/themes/twentyfifteen/js/skip-link-focus-fix.js

```diff
@@ -1,21 +1,26 @@
 /**
- * Makes "skip to content" link work correctly in IE9, Chrome, and Opera
- * for better accessibility.
+ * File skip-link-focus-fix.js.
  *
- * @link http://www.nczonline.net/blog/2013/01/15/fixing-skip-to-content-links/
+ * Helps with accessibility for keyboard only users.
+ *
+ * Learn more: https://git.io/vWdr2
  */
-
 ( function() {
-	var ua = navigator.userAgent.toLowerCase();
-
-	if ( ( ua.indexOf( 'webkit' ) > -1 || ua.indexOf( 'opera' ) > -1 || ua.indexOf( 'msie' ) > -1 ) &&
-		document.getElementById && window.addEventListener ) {
+	var isIe = /(trident|msie)/i.test( navigator.userAgent );
 
+	if ( isIe && document.getElementById && window.addEventListener ) {
 		window.addEventListener( 'hashchange', function() {
-			var element = document.getElementById( location.hash.substring( 1 ) );
+			var id = location.hash.substring( 1 ),
+				element;
+
+			if ( ! ( /^[A-z0-9_-]+$/.test( id ) ) ) {
+				return;
+			}
+
+			element = document.getElementById( id );
```

Comment:
```suggestion
			var id = location.hash.substring( 1 );

			if ( ! ( /^[A-z0-9_-]+$/.test( id ) ) ) {
				return;
			}

			var element = document.getElementById( id );
```
Minor suggestion

Reply:
Thanks for the suggestion. It feels awkward to make code quality changes to a script when deprecating it, though I already adjusted some spacing in Twenty Fifteen and Twenty Sixteen when I copied the code from Twenty Seventeen.

If I understand the [coding standards document](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#declaring-variables-with-var) correctly, would declaring these variables in the original `var` line be better?
```
( function() {
	var id, element,
		isIe = /(trident|msie)/i.test( navigator.userAgent );
```

You could continue the conversation here if you prefer, but I should create a new PR in the main repository.